### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bjaspan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+TEST EDIT
```
Comment (bjaspan) [inline README.md:1]: You misspelled EDIT.

Comment (dminca) [inline README.md]: LGTM :+1:

**Review by dminca — APPROVED** [review]
LGTM :+1:

**Comment by bjaspan** [comment]
@dminca I was demonstrating pull requests for someone who has never used GitHub. :-)

Are you using Goofy? What brings you here?

**Comment by dminca** [comment]
@bjaspan I was playing with the new Review feature :smile: 

Yes, I've played with it a little, tested it with a docker swarm (v1.12) and it's a very good stress-testing tool I can say. :+1:

**Comment by bjaspan** [comment]
Glad to hear it! 

I was unaware that anyone was using Goofy. I'm happy to know that someone is. :-)

**Comment by dminca** [comment]
Your repo has also been mentioned in the [DevOps Links #25](http://us3.campaign-archive1.com/?u=29dd5291078113789eaec5a6b&id=f2fc670eea&e=a51317aa23) newsletter, but I'm guessing you knew that :smile:

**Comment by bjaspan** [comment]
I did not know that! Thanks.

**Pull request closed without merging**